src/applications/vaos/tests/e2e/page-objects/AudiologyPageObject.js

```diff
@@ -9,10 +9,11 @@ export class AudiologyPageObject extends PageObject {
   }
 
   selectTypeOfCare(label) {
```

Comment:
This would be a good method to add to the PageObject.

```
selectRadioButtonShadow({label}) {
  cy.get('va-radio')
      .shadow()
      .get('va-radio-option')
      .contains(label)
      .click();

    return this;
}
```

Reply:
Good point, let me refactor.